cloudinit/config/cc_ubuntu_advantage.py

```diff
@@ -147,6 +148,9 @@
     "ua_apt_https_proxy",
 )
 
+# Pro cli message codes
+_PRO_MC_SERVICE_ALREADY_ENABLED = "service-already-enabled"
```

Comment:
Maybe it's worth us `from uaclient.messages impor ALREADY_ENABLED` and matching `ALREADY_ENABLED.name` in our errors checks?

Reply:
Done.